docs/rabbitmq.config.example

```diff
@@ -333,18 +370,76 @@
    %%
    %% {mnesia_table_loading_retry_timeout, 30000},
 
-   %% Size in bytes below which to embed messages in the queue index. See
-   %% http://www.rabbitmq.com/persistence-conf.html
+   %% Size in bytes below which to embed messages in the queue index.
+   %% See http://www.rabbitmq.com/persistence-conf.html
    %%
    %% {queue_index_embed_msgs_below, 4096},
 
+   %% Maximum number of queue index entries to keep in journal
+   %% See http://www.rabbitmq.com/persistence-conf.html
+   %%
+   %% {queue_index_max_journal_entries, 32768},
+
+   %% Number of credits that a queue process is given by the message store
+   %% By default, a queue process is given 4000 message store credits,
+   %% and then 800 for every 800 messages that it processes.
+   %%
+   %% {msg_store_credit_disc_bound, {4000, 800}},
+
+   %% Minimum number of messages with their queue position held in RAM required
+   %% to trigger writing their queue position to disk.
+   %%
+   %% This value MUST be higher than the initial msg_store_credit_disc_bound value,
+   %% otherwise paging performance may worsen.
+   %%
+   %% {msg_store_io_batch_size, 4096},
+
+   %% Number of credits that a connection, channel or queue are given
+   %% By default, every connection, channel or queue is given 400 credits,
+   %% and then 200 for every 200 messages that it processes
+   %% See https://www.rabbitmq.com/blog/2015/10/06/new-credit-flow-settings-on-rabbitmq-3-5-5/
```

Comment:
We should mention that increasing these values may help with throughput but is dangerous (very high values are no different from having any flow control).